src/Pages/Contact/index.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
```

Comment:
Checkout button should not be clickable if no ingridients

<img width="856" alt="Screenshot 2023-01-09 at 21 40 17" src="https://user-images.githubusercontent.com/34886027/211393684-9a77c0c8-42cb-4ad3-8414-d66b489da135.png">